frontend/src/utils/generalUtils.ts

```diff
@@ -135,6 +135,35 @@ export const validateSubmitTxInputs = (
   }
 }
 
+export const validateSubmitTxInputsSkyline = (
+  settings: ISettingsState, sourceChain: ChainEnum, destinationChain: ChainEnum,
+  destinationAddr: string, amount: string, bridgeTxFee: string, isNativeToken: boolean
+): string | undefined => {
+  if (BigInt(amount) < BigInt(settings.minValueToBridge)) {
+    return `Amount less than minimum: ${convertUtxoDfmToApex(settings.minValueToBridge)} APEX`;
+  }
+
+  const maxAllowedToBridgeDfm = BigInt(settings.maxAmountAllowedToBridge)
```

Comment:
wrap this whole maxAllowedToBridge thing in if (!isNativeToken).
basically, this validation should only be executed if !isNativeToken